src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -0,0 +1,85 @@
+import { fhirBaseUrl, openmrsFetch, Session } from "@openmrs/esm-framework";
+import dayjs from "dayjs";
+import useSWR from "swr";
+import { MedicationDispense, MedicationRequest, OrderConfig } from "../types";
+
+export function saveMedicationDispense(
+  medicationDispense: MedicationDispense,
+  abortController: AbortController
+) {
+  return openmrsFetch(`${fhirBaseUrl}/MedicationDispense`, {
+    method: "POST",
+    signal: abortController.signal,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: medicationDispense,
+  });
+}
+
+export function useOrderConfig() {
+  const { data, error, isValidating } = useSWR<{ data: OrderConfig }, Error>(
+    `/ws/rest/v1/orderentryconfig`,
+    openmrsFetch
+  );
+  return {
+    orderConfigObject: data ? data.data : null,
+    isLoading: !data && !error,
+    isError: error,
+    isValidating,
+  };
+}
+
+export function initiateMedicationDispenseBody(
+  medicationRequests: Array<MedicationRequest>,
+  session: Session
+) {
+  let dispenseBody = [];
+  medicationRequests.map((medicationRequest) => {
+    let dispense = {
+      resourceType: "MedicationDispense",
+      status: "on-hold", // might need to change this to appropriate status
+      medicationReference: medicationRequest.medicationReference,
+      subject: medicationRequest.subject,
+      performer: [
+        {
+          actor: {
+            reference: medicationRequest.requester.reference,
```

Comment:
This shouldn't default to the requester, as the the person who dispensing the medication will almost certainly be someone different than the person prescribing it... this would likely default to the provider associated with the current user session, if that's available.  Otherwise could be left empty for now.

Reply:
This was an oversight on my end.